README.md

```diff
@@ -106,3 +106,11 @@ Templates:
 
 - [testingjavascript.txt](templates/testingjavascript.txt)
 
+### [Vim for VSCode](https://vimforvscode.com) ⌨️
+
+Learn how to use Vim with VSCode
+
+Templates:
+
+- [vim-for-vscode.txt](templates/vim-for-vscode.txt)
```

Comment:
Awesome! I'm guessing you're working on creating this template?

Reply:
Either you commented on this before it was added, or I misunderstood 😛

I added it here -> [templates/vim-for-vscode.txt](https://github.com/kentcdodds/kcd-learning-clubs-ideas/blob/2ca4607c36082333c87ecf590cfeedf860d2c208/templates/vim-for-vscode.txt)